### ISSUE TITLE
Channel Switches triggering Aqara Alarm

### ISSUE THREAD

**Issue by Crox77** [AdamSwinden/homebridge-sky-q#11]
Bit of a random one and really odd but after some advice please?
I’ve added a few channel switches and they work fine no problem when selecting them in the Apple Home App. But when I say hey Siri to my HomePod followed by the channel I want it changes the channel no problem but it Arms my Aqara Alarm! I have no automations linking these and it’s only started doing it in the last couple of days. Anybody have any ideas please?
Thanks Paul

**Comment by leoneleone** [comment]
What is the name of the channel switch in HomeKit? 
And what is the exact phrase you are using with Siri? 

And what iOS version is your HomePod running?

**Comment by Crox77** [comment]
Hi it does it with any Sky sports channels and did it with bbc one this morning.

HomePod iOS 13.2.1

Hey Siri Sky Sports news on

My channels are 
Sky Sports News
Sky Sports F1
Sky Sports Football
Sky Premier League
BBC One
BBC 2
ITV
Channel 4
Channel 5
Hope this helps thanks

**Comment by leoneleone** [comment]
Can you try: 
“Hey Siri, turn on the Sky Sports News switch” ?

Do you get the same problem?

**Comment by Crox77** [comment]
So I tried hey Siri Turn on sky sports news and it didn’t do it! 
Any ideas why it was doing it?

**Comment by Crox77** [comment]
Thanks for your help

**Comment by leoneleone** [comment]
You’re welcome 🤓

I have an extensive HomeKit setup and use Siri Voice Commands to do most things, and yes I’ve encountered this kind of situation before usually around new iOS software updates. 

My advice would be to look at the naming of your devices, both your ‘channel switches’ and the Aqara Alarm/Hub. Use an app like the Eve.app so you can see both the ‘Accessory name’ and the ‘Siri name’ for each Device/Accessory. Cleaning up and streamlining you naming scheme can help. 

As to why your Aqara Alarm keeps getting armed... my guess is it’s a bug particular to Siri Voice Commands in HomeKit. 
It could be a bug in your particular HomeKit setup, and only a complete reset of your HomeKit configuration would clear that up. Always a last resort fix for me. 

 Support are almost useless when it comes to HomeKit issues such as these, because they do not have a dedicated HomeKit Support team (still!). I would submit a bug report through the Apple Feedback page or iOS Support app all the same. 

Good luck 😊

**Comment by Crox77** [comment]
Top man thanks again, odd thing though it worked for a couple days fine

**Comment by leoneleone** [comment]
Yes. I’ve long suspected that  updates the Siri Commands handling server-side rather than via software updates, but I have no proof of this -  just a hunch. Cleaning up your naming scheme is good practice all the same.

You know, you can also try: 
“Hey Siri, Sky Sports News” - the name of the switch(channel) without the word ‘on’.

Simple saying the ‘Siri name’ of any switch will turn that switch on. You have to use the word ‘off’ to actually turn any switch off. 

🤓
